docs/producers-API-documentation.md

```diff
@@ -0,0 +1,283 @@
+# Block Producers Data
+
+## Overview
+
+The API allows getting the data from block producers of different EOSIO + Antelope blockchain networks, available on [Antelope Tools](https://antelope.tools/) such as eos and jungle4 testnet. Among the data that can be obtained is the [BP JSON](https://github.com/eosrio/bp-info-standard) of each BP. This is useful because the logic to get all these objects is not needed to implement from scratch. Also, in the Antelope Tools backend, each API endpoint is requested through HTTP, and the obtained status is saved, which allows to obtain the status code of the request.
```

Comment:
...allows getting important data on block producers on the different EOSIO + Antelope blockchain networks available on [Antelope Tools](https://antelope.tools/) such as EOS and jungle4 testnet.